Test/SearchEngineTests.cs

```diff
@@ -140,7 +140,7 @@ public static void TestClassicSearchXcorrWithToml()
 
             List<PsmFromTsv> parsedPsms = PsmTsvReader.ReadTsv(psmFile, out var warnings);
 
-            Assert.AreEqual(127, parsedPsms.Count);
+            Assert.AreEqual(37, parsedPsms.Count);
             Assert.AreEqual(0, warnings.Count);
         }
```

Comment:
Should you load back in the results.txt to check the lines where it states how many 1%  PSM's there are to test if it is the correct value?